docs/ext/web.rst

```diff
@@ -118,7 +118,7 @@ To install, run::
 Mopidy-MusicBox-Webclient
 =========================
 
-https://github.com/woutervanwijk/Mopidy-MusicBox-Webclient
+https://github.com/pimusicbox/Mopidy-MusicBox-Webclient
```

Comment:
Even though there's not redirect happening, the repo name itself was lowercased when the repo moved to the new pimusicbox GitHub org. `https://github.com/pimusicbox/mopidy-musicbox-webclient` is the correct URL.